barretenberg/cpp/src/barretenberg/vm/avm/trace/common.hpp

```diff
@@ -52,11 +52,13 @@ static const uint32_t MAX_MEM_TAG = MEM_TAG_U128;
 enum class AvmError : uint32_t {
```

Comment:
Could we move the error to it's own file so that we don't have to include common.hpp (a bag of things) from addressing_mode.hpp?

Reply:
Done.